main.go

```diff
@@ -289,6 +290,17 @@ func (cfg *config) doProxy(w http.ResponseWriter, r *http.Request) {
 	h.ServeHTTP(w, r)
 }
 
+func (cfg *config) listModules(w http.ResponseWriter, r *http.Request) {
+	log.Debugf("Listing modules")
```

Comment:
It might be better to use html/template here to properly escape the string content being injected.